### PR TITLE
Rsvp model

### PR DESCRIPTION
**Pull request by exploreshaifali**

### CODE REVIEW

File: systers_portal/meetup/models.py [hunk 1]
```diff
@@ -45,3 +45,13 @@ def clean_fields(self, *args, **kwargs):
         """Validate meetup date is not less than today's date"""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+
+
+class Rsvp(models.Model):
+    """ Users RSVP for particular meetup """
+    user_id = models.ForeignKey(SystersUser)
+    meetup_id = models.ForeignKey(Meetup)
+    is_coming = models.BooleanField()
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: We also need to add a "plus one" field.

File: systers_portal/meetup/models.py [hunk 2]
```diff
@@ -45,3 +45,13 @@ def clean_fields(self, *args, **kwargs):
         """Validate meetup date is not less than today's date"""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+
+
+class Rsvp(models.Model):
+    """ Users RSVP for particular meetup """
+    user_id = models.ForeignKey(SystersUser)
+    meetup_id = models.ForeignKey(Meetup)
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: Just `meetup`.

File: systers_portal/meetup/models.py [hunk 3]
```diff
@@ -45,3 +45,13 @@ def clean_fields(self, *args, **kwargs):
         """Validate meetup date is not less than today's date"""
         if self.date < timezone.now().date():
             raise ValidationError("Date should not be less than today's date.")
+
+
+class Rsvp(models.Model):
+    """ Users RSVP for particular meetup """
+    user_id = models.ForeignKey(SystersUser)
```
Comment (ana-balica) [inline systers_portal/meetup/models.py]: Just 'user'.

File: systers_portal/meetup/models.py (hunk 1 above)
Comment (exploreshaifali) [inline systers_portal/meetup/models.py]: Sorry but I am not clear why we need `plus_one` attribute. If user say "yes" for is_coming that implicitly means `plus_one`.

Comment (ana-balica) [inline systers_portal/meetup/models.py]: I assumed that `is_coming` means if the user himself/herself is coming to the meetup.
`plus_one` means that the user is coming and also bringing one extra person with him/her.

Comment (exploreshaifali) [inline systers_portal/meetup/models.py]: Okay, but what if user want to bring more than one person with him/her?

Comment (ana-balica) [inline systers_portal/meetup/models.py]: Then those extra 10 people maybe should register themselves and rsvp.
It's just a common thing to let people have +1 (at least in European/American cultures). I assume it began as inviting your spouse/SO. Check the meetup.com. Plus we have it in the requirements doc (I think).

File: systers_portal/meetup/models.py (hunk 2 above)
Comment (exploreshaifali) [inline systers_portal/meetup/models.py]: Done.

File: systers_portal/meetup/models.py (hunk 3 above)
Comment (exploreshaifali) [inline systers_portal/meetup/models.py]: Done.

File: systers_portal/meetup/models.py (hunk 1 above)
Comment (exploreshaifali) [inline systers_portal/meetup/models.py]: Done :)

**Comment by ana-balica** [comment]
Merged with some additional commits :)
Don't forget to fetch from develop!

**Pull request closed without merging**